sdk/resourcemanager/azure-resourcemanager-containerinstance/src/main/java/com/azure/resourcemanager/containerinstance/implementation/ContainerGroupImpl.java

```diff
@@ -445,6 +446,14 @@ public ContainerGroupImpl withDnsPrefix(String dnsPrefix) {
         return this;
     }
 
+    @Override
+    public ContainerGroupImpl withAutoGeneratedDomainNameLabelScope(DnsNameLabelReusePolicy reusePolicy) {
+        if (reusePolicy != null) {
+            this.innerModel().ipAddress().withAutoGeneratedDomainNameLabelScope(reusePolicy);
+        }
```

Comment:
make sure ipAddress is not null

https://github.com/Azure/azure-sdk-for-java/blob/6e748a33351eeda68205b4a03d69db268dd3d0ed/sdk/resourcemanager/azure-resourcemanager-containerinstance/src/main/java/com/azure/resourcemanager/containerinstance/implementation/ContainerGroupImpl.java#L441-L443